Radio/example.jsx

```diff
@@ -99,6 +105,13 @@ export default {
         value='ipsum'
       />,
 
+      'Partially collapsed & controlled': <Radio
+        options={optionsWithContent}
+        expandLabel='Show me all the options'
+        collapsedOptions={2}
```

Comment:
Sorry, what I mean to imply is that some other changes I will do in a PR targeting this PR, so that we can safely merge this one and release of deadline so demands it.